assets/queries/cloudFormation/aws_sam/serverless_api_without_content_encoding/query.rego

```diff
@@ -0,0 +1,57 @@
+package Cx
+
+import data.generic.common as common_lib
+
+CxPolicy[result] {
+	document := input.document
+	resource = document[i].Resources[name]
+	resource.Type == "AWS::Serverless::Api"
+	properties := resource.Properties
+
+	properties.MinimumCompressionSize < 0
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("Resources.%s.Properties.MinimumCompressionSize", [name]),
+		"issueType": "IncorrectValue",
+		"keyExpectedValue": "Resources.%s.Properties.MinimumCompressionSize is greater than -1 and smaller than 10485760",
+		"keyActualValue": "Resources.%s.Properties.MinimumCompressionSize is set but smaller than 0",
+		"searchLine": common_lib.build_search_line(["Resources", name, "Properties", "MinimumCompressionSize"], []),
+	}
+}
+
+CxPolicy[result] {
+	document := input.document
+	resource = document[i].Resources[name]
+	resource.Type == "AWS::Serverless::Api"
+	properties := resource.Properties
+
+	properties.MinimumCompressionSize > 10485759
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("Resources.%s.Properties.MinimumCompressionSize", [name]),
+		"issueType": "IncorrectValue",
+		"keyExpectedValue": "Resources.%s.Properties.MinimumCompressionSize is greater than -1 and smaller than 10485760",
+		"keyActualValue": "Resources.%s.Properties.MinimumCompressionSize is set but greater than 10485759",
+		"searchLine": common_lib.build_search_line(["Resources", name, "Properties", "MinimumCompressionSize"], []),
+	}
+}
```

Comment:
Line 11 and line 29 can be put in a function and both CxPolicies merged into one